app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -118,10 +119,16 @@ def redhat_provider
                        :resource_name => "providers")
     end
 
+    api :GET, '/activation_keys/:id/service_levels', N_('List available service levels for an activation key')
```

Comment:
I am still less of a fan of this given it leads to false pretense in my opinion that the levels are somehow scoped to the key being passed in, but if this is what users want then so be it.